corehq/apps/domain/forms.py

```diff
@@ -622,8 +626,10 @@ def __init__(self, *args, **kwargs):
         self.helper[5] = twbscrispy.PrependedText('hipaa_compliant', '')
         self.helper[6] = twbscrispy.PrependedText('two_factor_auth', '')
         self.helper[7] = twbscrispy.PrependedText('strong_mobile_passwords', '')
+        self.helper[8] = twbscrispy.PrependedText('ga_opt_out', '')
 
         if not domain_has_privilege(domain, privileges.ADVANCED_DOMAIN_SECURITY):
+            self.helper.layout.pop(8)
```

Comment:
Nitpick: it'd be nice to check `settings.ANALYTICS_IDS` to see if GA is even on the environment.